index.html

```diff
@@ -5,8 +5,9 @@
     <title>Lambda School Newsfeed</title>
     <link rel="stylesheet" type="text/css" media="screen" href="./CSS/index.css" />
     <script src="./Menu/Menu.js"></script>
```

Comment:
Let's not forget to add 'async' (preferably) or 'defer' if you have a specific use case for our JS scripts.